okhttp-logging-interceptor/src/test/java/okhttp3/logging/HttpLoggingInterceptorTest.java

```diff
@@ -38,14 +38,13 @@
 import mockwebserver3.MockResponse;
 import mockwebserver3.MockWebServer;
 import okhttp3.tls.HandshakeCertificates;
-import okio.Buffer;
-import okio.BufferedSink;
-import okio.ByteString;
+import okio.*;
```

Comment:
Fix your IDE settings?

Reply:
I swear I set this all the time.  import square code style.  :)